volunteer_categories/admin.py

```diff
@@ -26,6 +48,71 @@ class RequestAdmin(admin.ModelAdmin):
         "status",
     ]
 
+    change_list_template = "admin/request_changelist.html"
+
+    def get_urls(self):
+        urls = super().get_urls()
+        my_urls = [
+            path("export-checkin/", self.export_checkin),
+        ]
+        return my_urls + urls
+
+    def export_checkin(self, request):
+        if request.method == "POST":
+            form = DailyCheckinForm(request.POST)
+            # check whether it's valid:
+            if form.is_valid():
+                selected_date = form.cleaned_data["selected_date"]
+                accepted_requests = Request.requests.select_related(
+                    "user", "role__category"
+                ).filter(
+                    role__category__start_time__date=selected_date, status="ACCEPTED"
+                )
+                if len(accepted_requests) != 0:
+
+                    field_names = [
+                        "last_name",
+                        "first_name",
+                        "start_time",
+                        "end_time",
+                        "role",
+                        "signature",
+                    ]
+
+                    response = HttpResponse(content_type="text/csv")
+                    response[
+                        "Content-Disposition"
+                    ] = "attachment; filename=daily-checkin-{}.csv".format(
+                        selected_date
+                    )
+                    writer = csv.writer(response)
+
+                    writer.writerow(field_names)
+                    for r in accepted_requests.order_by("user__last_name"):
+                        user = r.user
+                        role = r.role.category
+                        row = writer.writerow(
+                            [
+                                user.last_name,
+                                user.first_name,
+                                role.start_time.astimezone(timezone(TIME_ZONE)).time(),
+                                role.end_time.astimezone(timezone(TIME_ZONE)).time(),
+                                role.title,
```

Comment:
@gillianpierce ideally we have the category title and role title here, so we can see all the bartenders etc